include/revng/Pipeline/Invokable.h

```diff
@@ -179,6 +179,10 @@ auto *concantMultiImp(const First *, const Rest *...Others) {
   }
 }
 
+inline std::tuple<> *concantMultiImp() {
```

Comment:
Put some details in the commit message.

Reply:
What do you think?

```
[Pipeline: fix pure (model only) analysis support](https://github.com/revng/revng/pull/232/commits/0ccccb8679336cec28cb325d32502f7113afc0ee) 

Because of a missing function overload, no container-less analysis
could have been created. This adds the said overload.
```